src/components/Links/Links.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '@edx/paragon';
+
+import './Links.scss';
+
+function Links(props) {
+  const { title, links } = props;
+
+  function handleLinkClick(e) {
+    e.preventDefault();
+    // TODO: exact functionality TBD
+  }
+
+  const linkItems = links.map(link => (
+    <li key={link.href}>
+      <Icon className={['fa', 'fa-file', 'text-link-icon']} />
+      <a href={link.href} onClick={handleLinkClick}>{link.title}</a>
+    </li>
+  ));
+
+  return (
+    <section className="links">
```

Comment:
One minor suggestion here is to include a `className` prop on the `Links` component, and use the `classNames` package to concatenate any custom classes passed into the component, e.g.:

```
const { className } = props;
<section className={classNames('links', className)}>
```

That would make it easy to add any custom styles to individual blocks of links (such as margin bottom) when we get this added to the sidebar.